generators/gae/templates/appengine-web.xml.ejs

```diff
@@ -30,6 +30,13 @@
     <system-properties>
         <property name="java.util.logging.config.file" value="WEB-INF/logging.properties"/>
     </system-properties>
+    <class-loader-config>
+        <!-- temporary workaround to make sure the app uses it's memcache/mail jars.!-->
+        <priority-specifier filename="cache-api-1.0.0.jar"/>
+        <priority-specifier filename="cache-api-1.1.0.jar"/>
+        <priority-specifier filename="javax-mail-1.5.6.jar"/>
+        <priority-specifier filename="javax.mail-1.6.2.jar"/>
```

Comment:
LGTM..
Curious: 
1/ we were not doing that before?
2/ do we really need to have the 2 versions? If folks use the latest jHispter, they should only have the new cache and mail jars, so 1 of them would only be needed?

Reply:
we had these before, but i removed it while testing w/ hazelcast thought we didn't need these anymore. my mistake :)

i don't think we need the other versions; removing them from this list.